src/uipath/_cli/_evals/_runtime.py

```diff
@@ -167,6 +170,18 @@ def __init__(
         self.logs_exporter: ExecutionLogsExporter = ExecutionLogsExporter()
         self.execution_id = str(uuid.uuid4())
 
+    def _initialize_input_schema(self) -> None:
+        """Initialize the input schema using a temporary runtime to get the entrypoint."""
+        temp_context = self.factory.new_context(
+            entrypoint=self.context.entrypoint, runtime_dir=os.getcwd()
+        )
+        temp_runtime = self.factory.from_context(temp_context)
+        self.context.input_schema = temp_runtime.get_entrypoint.input
```

Comment:
If we make `get_entrypoint` async, you can execute it in `execute` and then pass the input schema (I would actually pass the whole `Entrypoint` object) as method parameters, instead of caching it on `self.context`